docs/ci-setup.md

```diff
@@ -58,21 +58,15 @@ mkdir -p data/kbs-storage/default/image-kek
 echo LieOhvkqFcGMzZrVzt6vPWlj/F/bgYMNe45vhQpdxAA= | base64 -d > data/kbs-storage/default/image-kek/11032d96-dccd-46a3-9244-93644d76745f
 ```
 # Github Runner Service
-For `enclave-cc` e2e tests, we run a "job-started" pre-cleanup job configured
-for the runner:
-
-```
-Environment=ACTIONS_RUNNER_HOOK_JOB_STARTED=<path/to>/job-started-cleanup.sh
-```
+For `enclave-cc` e2e tests, we run a "post-action" job to clean the useless files generated by the CI.
```

Comment:
I don't remember why I used `ACTIONS_RUNNER_HOOK_JOB_STARTED` but if it needs to be a "post-action", then the built-in mechanism via `ACTIONS_RUNNER_HOOK_JOB_COMPLETED` is preferred rather than using `e2e.yaml` for this.

Reply:
Yes. I think post-action should be better. This would make every ci pipeline idempotent, without leaving anything left on the machine.

Reply:
I think this would let all jobs on the runner to run this clean up script. this is not expected. As the runner is shared among multiple repos, different repo would have different steps of clean jobs.

Reply:
That would be my expectation and that's what we'd end up doing with this change also: both `guest-components` and `enclave-cc` end up running `${HOME}/script/post_action.sh`. `GITHUB_REPOSITORY` variable can be used to run repo specific commands.

Reply:
Ok I think it deserves a try.